clients/fides-js/src/components/ConsentContent.tsx

```diff
@@ -4,7 +4,7 @@ import { ExperienceConfig } from "../lib/consent-types";
 
 import GpcInfo from "./GpcInfo";
 import ExperienceDescription from "./ExperienceDescription";
-import { getConsentContext } from "../fides";
+import { getConsentContext } from "../lib/consent-context";
```

Comment:
this import was causing a warning in the build step. I guess it prefers the full path relative import